sdk/nodejs/src/client.ts

```diff
@@ -69,6 +71,8 @@ export class DevCycleClient {
     }
 
     constructor(sdkKey: string, options?: DevCycleServerSDKOptions) {
+        // generate UUID for this client
+        this.clientUUID = uuidv4()
```

Comment:
do we really need to add a dependency just to generate a unique id?
we could just do something like this?
https://stackoverflow.com/a/40191779

Reply:
yea makes sense.